assets/sass/components/analytics-4/audience-segmentation/_googlesitekit-audience-selection-panel.scss

```diff
@@ -22,10 +22,6 @@
 
 		.googlesitekit-selection-box {
 
-			.mdc-checkbox__description {
-				padding-top: 4px;
```

Comment:
Only ting removed from audience segmentation style to match the spacing between content for consistency